lib/modules/datasource/docker/index.spec.ts

```diff
@@ -219,7 +255,7 @@ describe('modules/datasource/docker/index', () => {
         .get('/')
         .reply(401, '', {
           'www-authenticate':
-            'Bearer realm="https://auth.docker.io/token",service="registry.docker.io",scope="repository:samalba/my-app:pull  "',
+            'Bearer realm="https://auth.docker.io/token",service="registry.docker.io",scope="repository:library/some-dep:pull  "',
```

Comment:
Can you please revert those unrelated naming changes? Or are the tests wrong?

Reply:
The code I have added use the scope provided in 401 response.
I had to either change this line or  line 265 `&scope=repository:library/some-dep:pull'` or this one.
they must be aligned.